_i18n/de.yml

```
@@ -0,0 +1,90 @@
+meta:
+  description: Entwickle 2D- und 3D-Spiele, plattformübergreifenden Projekte oder sogar XR-Ideen!
+header:
+  features: Features
+  showcase: Vorzeigeprojekte
+  blog: Blog
+  community: Gemeinschaft
+  assets: Assets
+  download: Download
+  docs: Docs
+  contribute: Mitwirken
+  donate: Spenden
+  language: Sprache
+footer:
+  download: Download
+  documentation: Dokumentation
+  features: Features
+  web_editor: Web Editor
+  release_archive: Release Archiv
+  source_code: Quellcode
+  project: Projekt
+  blog: Neuigkeiten
+  code_of_conduct: Verhaltenskodex
+  governance: Verwaltung
+  teams: Teams
+  priorities: Prioritäten
+  communities: Gemeinschaften
+  resources: Ressourcen
+  asset_library: Asset-Bibliothek
+  press_kit: Presse-Kit
+  showcase: Vorzeigeprojekte
+  education: Bildung
+  foundation: Stiftung
+  about: Über uns
+  donate: Spenden
+  license: Lizenz
+  privacy_policy: Datenschutzerklärung
+  contact_us: Kontakt
+  text: "Juan Linietsky, Ariel Manzur und <a href='https://github.com/godotengine/godot/blob/master/AUTHORS.md' target='_blank' rel='noopener'>Beitragende</a>. Gehostet von der <a href='https://godot.foundation/' target='_blank' rel='noopener'>Godot Foundation</a>. Website Quellcode <a href='https://github.com/godotengine/godot-website' target='_blank' rel='noopener'>auf  GitHub</a>."
+sponsors:
+  title: "Godot wird gesponsert durch:"
+  text: Wenn Sie an einem Unternehmensponsoring interessiert sein, schreiben Sie bitte an <a href='mailto:contact@godot.foundation'>contact@godot.foundation</a>.<br>Spenden als Einzelpersonen können durch den <a href='https://fund.godotengine.org/'>Godot Engine Entwicklungsfonds</a> getätigt werden.
```

Meta and other places here uses *Du*, this *Sie*. We need to decide on one for all german comms or at least the website+docs, IMO.

---

Yeah this is one reason why I wanted feedback first. 

The "Du" will look more casual for hobbyists while "Sie" will IMHO feel too distant. I would personally go with "Du" but we chose to go with "Sie" in the docs...

---

The games industry tends to lean towards casual as a whole. So I personally would go with "Du" - it also often comes with shorter and easier to understand language in tow, which is more accessible. Especially when considering who relies on the translation to begin with.

---

```suggestion
  text: Um Unternehmens-Sponsor zu werden, einfach eine Nachricht an <a href='mailto:contact@godot.foundation'>contact@godot.foundation</a> schreiben.<br>Individuelle Spenden sind durch die<a href='https://fund.godotengine.org/'>Godot Engine Entwicklungsfonds</a> möglich.
```
Some word magic to make it sound more natural and remove the problem alltogether